examples/nginx.conf

```diff
@@ -107,6 +108,26 @@ http {
             }
         }
 
+        location /suffix {
```

Comment:
Please add comments here about why we need this location and how this routing rule works.